mlflow/xgboost.py

```diff
@@ -152,14 +152,14 @@ def save_model(
 
     # Save an XGBoost model
     xgb_model.save_model(model_data_path)
-
+    xgb_model_class = ".".join([xgb_model.__module__, xgb_model.__class__.__name__])
```

Comment:
Can we use `_get_fully_qualified_class_name` here?

https://github.com/mlflow/mlflow/blob/4195541c7f6a79fba4ccad2295534f16eaa8db39/mlflow/utils/__init__.py#L126-L130

Reply:
Thanks for your suggestion! It is what I was looking for :laughing: